File: NewReview/newsreview/src/main/kotlin/com/newsreview/model/Article.kt
package com.newsreview.model

import org.springframework.data.annotation.Id
import org.springframework.data.relational.core.mapping.Table

@Table("articles")
data class Article(
    @Id
    val id: Long? = null,
    val newsId: String,
    val title: String,
    val content: String,
    val authorName: String
) 

File: NewsSite/app/models/article.py
from datetime import datetime
from app import db

class Article(db.Model):
    __tablename__ = 'articles'
    
    id = db.Column(db.Integer, primary_key=True)
    news_id = db.Column(db.String(50), unique=True)
    title = db.Column(db.String(500))
    content = db.Column(db.Text)
    author_name = db.Column(db.String(255))
    publish_time = db.Column(db.DateTime, nullable=False)
    
    def __repr__(self):
        return f'<Article {self.title}>' 